src/Runner.Worker/Container/ContainerInfo.cs

```diff
@@ -260,18 +260,27 @@ private void UpdateWebProxyEnv(RunnerWebProxy webProxy)
 
     public class MountVolume
     {
+        public string UserProvidedValue { get; set; }
```

Comment:
Naming suggestions welcome

Reply:
I like it, to be honest. It is the string value user provided after all :relaxed:

Reply:
Should we just maintain a list of the system provided mounts and do .Where not in that list?